x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/factories/wrap_hits_factory.ts

```diff
@@ -38,6 +38,7 @@ export const wrapHitsFactory =
         _id: id,
         _index: '',
         _source: {
+          // TODO: executionId here as well?
```

Comment:
Not necessary, already added higher up as wrapped by [create_persistence_rule_type_wrapper](https://github.com/elastic/kibana/blob/ccb49e5676de92c5968d9ca4f4be9404afc1816d/x-pack/plugins/rule_registry/server/utils/create_persistence_rule_type_wrapper.ts#L87).